azure-cognitiveservices-vision-computervision/MANIFEST.in

```diff
@@ -1 +1 @@
-include *.rst
+include *.md
```

Comment:
This will exclude the HISTORY.rst, no good :(

Reply:
@lmazuel If we convert HISTORY to *.md, is this change OK?

Cc: @diberry

Reply:
@mmacy @diberry Laurent is out this week. Yes. That will work. Then on line 55, concat on that new history.md as well. (laurent has a comment on that as well)